src/components/implementation/no_interface/cFE_booter/osfilesys.h

```diff
@@ -1,113 +1,258 @@
-#include "cFE_util.h"
 #include <string.h>
+#include <dirent.h>
+
 #include "gen/common_types.h"
 #include "gen/osapi.h"
 #include "gen/osapi-os-filesys.h"
 
+#include <cos_component.h>
+#include <cos_defkernel_api.h>
+#include <cos_kernel_api.h>
+
+#include "cFE_util.h"
+
 // Not to be confused with similar OSAL constants.
 // These are only to define size of statically allocated data
-#define MAX_NUM_FS 3
+#define MAX_NUM_FS NUM_TABLE_ENTRIES
 #define MAX_NUM_FILES 100
+#define MAX_NUM_DIRENT 10
 
-#define TAR_BLOCKSIZE 512
-#define INT32_MAX 0x7FFFFFF //2^31 - 1
+// a page is 4096, size of f_part is 5 values * 4 bytes
+#define F_PART_DATA_SIZE (4096-sizeof(struct f_part))
```

Comment:
did you mean PAGE_SIZE? PAGE_SZ isnt used.  also is it worth including shared consts for just that and does the bump pointer always return 4096 or does it return PAGE_SIZE